bridges/centralized-ethereum/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "witnet-centralized-ethereum-bridge"
+version = "0.1.0"
+authors = ["Witnet Foundation <info@witnet.foundation>"]
+edition = "2018"
+
+[dependencies]
+actix = { version = "0.11.0-beta.1", default-features = false }
```

```suggestion
actix = { version = "0.11.1", default-features = false }
```

After #1902 is merged